src/deepsparse/server/openai_server.py

```diff
@@ -53,17 +52,132 @@
 
 
 class OpenAIServer(Server):
+    def __init__(self, **kwargs):
+        self.model_list = ModelList()
+        self.model_to_pipeline = {}
+
+        super().__init__(**kwargs)
+
     def _add_routes(self, app: FastAPI):
         for endpoint_config in self.server_config.endpoints:
-            self._add_endpoint(
+            self._add_model(
                 app,
                 endpoint_config,
             )
 
-        _LOGGER.info(f"Added endpoints: {[route.path for route in app.routes]}")
+        app.model_list = self.model_list
+        app.model_to_pipeline = self.model_to_pipeline
+
+        @app.get("/v1/models", tags=["model"])
+        async def show_available_models():
+            """Show available models. Right now we only have one model."""
```

Comment:
> Right now we only have one model.

Is this true if we can load multiple models with one config?

Reply:
Good catch, updating.